internal/config/config.go

```diff
@@ -161,3 +161,8 @@ func Load(path string, opts *LoadOptions) (*Config, error) {
 func (c *Config) HCLContext() *hcl.EvalContext {
 	return c.ctx.NewChild()
 }
+
+// GetConfigPath returns the path to the directory that contains the config file (waypoint.hcl)
+func (c *Config) GetConfigPath() string {
```

Comment:
Getters usually omit the `Get`, recommend just `ConfigPath() string` here.

Ref: https://go.dev/doc/effective_go#Getters